lib/debugger/debugger.c

```diff
@@ -201,6 +202,32 @@ _cmd_quit(Debugger *self, gint argc, gchar *argv[])
   return FALSE;
 }
 
+static gboolean
+_cmd_info_pipe(Debugger *self, LogPipe *pipe)
+{
+  gchar buf[1024];
+
+  printf("LogPipe %p at %s\n", pipe, log_expr_node_format_location(pipe->expr_node, buf, sizeof(buf)));
+  _display_source_line(pipe->expr_node);
+
+  return TRUE;
+}
+
+static gboolean
+_cmd_info(Debugger *self, gint argc, gchar *argv[])
+{
+  if (argc && strcmp(argv[1], "pipe") == 0)
```

Comment:
`argc` should be checked to be 2. The current implementation segfaults.

Reply:
nice catch. thanks.